sdk/samples/iot/CMakeLists.txt

```diff
@@ -133,4 +133,20 @@ target_link_libraries(paho_iot_provisioning_sas_sample
     az::iot::samples::common
 )
 
-endif() # TRANSPORT_PAHO
+# Workaround for linker warning LNK4098: defaultlib 'LIBCMTD' conflicts with use of other libs
+if (MSVC)
+    set_target_properties(
+        paho_iot_hub_c2d_sample
+        paho_iot_hub_methods_sample
+        paho_iot_hub_pnp_sample
+        paho_iot_hub_telemetry_sample
+        paho_iot_hub_sas_telemetry_sample
+        paho_iot_hub_twin_sample
+        paho_iot_provisioning_sample
+        paho_iot_provisioning_sas_sample
+        PROPERTIES LINK_FLAGS
+        "/NODEFAULTLIB:libcmtd.lib"
+    )
+endif()
+
+endif() # TRANSPORT_PAHO
```

Comment:
nit: Needs a new line at eof.

Reply:
In this same file, you initially removed the last new line at eof, so I left it as you had it. See: https://github.com/Azure/azure-sdk-for-c/commit/df4b7e032abc8adda8e1c27026d2800fbd76c178#diff-08e085d86a6c448fc1ded13223c00a1f
You have also previously done a nit on their being a new line at the eof. I actually kept it off here because I figured you would nit it.
Moving forward, I plan to add a new line at the eof, regardless of what kind of file it is.

Reply:
Apologies, let me try to clear up the confusion.
We want **one** new line character at the end of the file, **not zero, not two**.

This file initially had **two**, so I removed **one of the two** in the diff you linked above, so we have **one**.

This PR removed another one and now we have **zero**.

You can see this if you look at the files view on GitHub, where we see the warning for the lack of newline:
https://github.com/Azure/azure-sdk-for-c/pull/985/files
![image](https://user-images.githubusercontent.com/6527137/89466801-5650fa80-d729-11ea-9969-682aa23ce91a.png)

> Moving forward, I plan to add a new line at the eof, regardless of what kind of file it is.

Right, this makes sense. Just don't add two :)

You don't want to have an extra new line beyond just ending the file with a **single** `\n` character. GitHub knows not to show that extra line in the file view. For example, open this file up in GitHub and IDEs and though there is **one** new line character at eof, which is what we want, IDEs show an extra line number and GitHub doesn't:
https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/src/azure/core/CMakeLists.txt#L86

**What editor/IDE are you using?**

In VSCode for example, file endings should look like this:
![image](https://user-images.githubusercontent.com/6527137/89467069-c8294400-d729-11ea-8b74-1cf8c6668b30.png)

And **not** like either of these:
![image](https://user-images.githubusercontent.com/6527137/89467096-d6776000-d729-11ea-9bf6-295eaa3425a2.png)
![image](https://user-images.githubusercontent.com/6527137/89467127-e8f19980-d729-11ea-96d0-f9aea86a1e60.png)